scipy/spatial/qhull.pyx

```diff
@@ -706,6 +720,38 @@ cdef class _Qhull:
     @cython.final
     @cython.boundscheck(False)
     @cython.cdivision(True)
+    def get_hull_points(self):
+        cdef vertexT *vertex
+        cdef int i, j, numpoints
+
+        self.check_active()
+
+        point_ndim = self.ndim
+
+        if self._is_halfspaces:
+            point_ndim -= 1
+
+        if self._is_delaunay:
+            point_ndim += 1
+
+        numvertices = self._qh.num_vertices
+
+        vertex = self._qh.vertex_list
+        points = np.zeros((numvertices, point_ndim), dtype=np.double)
+        i = 0
+        while vertex and vertex.next:
+            j = 0
+            for j in xrange(point_ndim):
+                points[i][j] = vertex.point[j]
```

Comment:
This is going to be somewhat slow --- points should be declared as `np.ndarray[np.npy_double, ndim=2]` and the indexing syntax is `points[i,j]`